pkg/drivemgr/drivemgr_service.go

```diff
@@ -66,3 +65,14 @@ func (svc *DriveServiceServerImpl) GetDrivesList(ctx context.Context, req *api.D
 		Disks: drives,
 	}, nil
 }
+
+// Locate invokes DriveManager's Locate method for manipulation drive's LED state
+func (svc *DriveServiceServerImpl) Locate(ctx context.Context, in *api.DriveLocateRequest) (*api.DriveLocateResponse, error) {
```

Comment:
Why do we need DriveLocateResponse?It's just a blank response?

Reply:
Interface is defined in [drivesvc.proto](https://github.com/dell/csi-baremetal/blob/feature-ISSUE-168-add-led-interface/api/v1/drivemgrsvc.proto#L27). Each rpc request should have response. That because we have empty struct here.